clang/test/CIR/IR/invalid-const-complex-wrong-real-type.cir

```diff
@@ -0,0 +1,11 @@
+// RUN: cir-opt %s -verify-diagnostics -split-input-file
+
+!s32i = !cir.int<s, 32>
```

Comment:
Thanks for adding the verifier tests. Please rename this file to `invalid-complex.cir` and incorporate the other test here as well. With `-split-input-file` you can test both in one single file.